core/storage/feedback/gae_models_test.py

```diff
@@ -0,0 +1,84 @@
+# coding: utf-8
+#
+# Copyright 2014 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and 
+# limitations under the License.
+
+__author__ = 'Shantanu Bhowmik'
+
+from core.platform import models
+(feedback_models,) = models.Registry.import_models([models.NAMES.feedback])
+import test_utils
+
+
+class SuggestionModelTest(test_utils.GenericTestBase):
+    """Tests the SuggestionModel class."""
+
+    def load_suggestions(self):
```

Comment:
Rather than creating a function called by each test, you can call this setUp and it will be called automatically before each test is called.